packages/storybook/stories/va-telephone.stories.jsx

```diff
@@ -2,28 +2,41 @@ import React from 'react';
 import {
   CONTACTS,
   contactsMap,
-  Table,
 } from '@department-of-veterans-affairs/component-library';
 import { getWebComponentDocs, propStructure, StoryDocs } from './wc-helpers';
 
 const telephoneDocs = getWebComponentDocs('va-telephone');
 
 // This builds the available "CONTACTS" list table
 // Descriptions are available in the contacts.js file
-const fields = [
+const columns = [
   { label: 'Property name (CONTACTS.x)', value: 'key' },
-  { label: 'Phone number', value: 'value' },
+  { label: 'Phone number', value: 'phone' },
   { label: 'Description', value: 'description' },
 ];
+const data = Object.entries(contactsMap).map(c => ([
+  c[0], c[1].phoneNumber, c[1].description
+]));
```

Comment:
Instead of an opaque array that's hard to 'grok' when rendering at L#31-36 below, I wonder if it would be more readable to use an object here?

Like this: `c => ({key: c[0], phoneNumber: c[1].phoneNumber, description: c[1].description})` and then below instead of the row.map, just spell out the individual elements like:

```
<va-table-row key={`row-${index}`}>
  <p key={`row-${index}-key`}>{row.key}</p>
  <p key={`row-${index}-phoneNumber`}>
    <va-telephone contact={row.phoneNumber}></va-telephone>
  </p>
  <p key={`row-${index}-description`}>{row.description}</p>
</va-table-row>
```